fuzzing/cc_deps.bzl

```diff
@@ -55,3 +47,19 @@ def cc_fuzz_test(
             name = name + "_corpus_zip",
             srcs = corpus,
         )
+        fuzzing_launcher(
+            name = name + "_run",
+            target = name,
+            corpus = name + "_corpus",
+            # Since the script depends on the _fuzz_test above, which is a cc_test,
+            # this attribute must be set.
+            testonly = True,
+        )
+    else:
+        fuzzing_launcher(
+            name = name + "_run",
+            target = name,
+            # Since the script depends on the _fuzz_test above, which is a cc_test,
+            # this attribute must be set.
+            testonly = True,
+        )
```

Comment:
We should avoid code duplication as much as possible. The fact that we even copy-pasted the exact code comments is another strong sign that this should be factored out.

How about instantiating `fuzzing_launcher` once and only setting the `corpus` attribute when needed, like this:

```
fuzzing_launcher(
  name = name + "_run",
  ...,
  corpus = name + "_corpus" if corpus else None,
)
```

Note the use of "None" to specify a `corpus` attribute that is not set - this is the way Starlark signals it.